panel/components/inspected-app/inspected-app.js

```diff
@@ -75,34 +75,34 @@ function inspectedAppService($rootScope, $q) {
     console.log(a);
   });
 
-  function onHintMessage(hint) {
-    if (hint.message) {
-      hints.push(hint);
-    } else if (hint.event) {
-      if (hint.event === 'hydrate') {
-        Object.keys(hint.data.scopes).forEach(function (scopeId) {
-          scopes[scopeId] = hint.data.scopes[scopeId];
+  function onHintMessage(message) {
```

Comment:
I assume the reason for `hint` -> `message` here is to make the property `isHint` more sensible, correct?